src/Util/RenameDuplicate.php

```diff
@@ -27,7 +27,7 @@ public function renameFile(string $filePathName) :string
         $fileName = $pathParts['filename'];
         $extension = $pathParts['extension'] ?? '';
 
-        $patterns = array('/^(.*)\((\d+)\)(\.?.*)$/','/(^((?!\.|\(\d+\)).)*$)()/');
+        $patterns = array('/^(.*)\((\d+)\)(\.?.*)$/','/(^(?:(?!\.|\(\d+\)).)*$)()/');
```

Comment:
Tested with Filename: "filedata-1.dat", on uploading a duplicate file. It renames the 2nd file to "filedata-1(2).dat", it should be "filedata-1(1).dat". The numbering of duplicate files should start from 1